Sources/main.cpp

```diff
@@ -50,6 +50,40 @@ static std::string gDefaultHost = "127.0.0.1";
 static bool gDaemonize = false;
 static bool gGDBCompat = false;
 
+#if defined(OS_POSIX)
+static void CloseFD() {
```

Comment:
These two functions seem unnecessary, you can just get rid of them and move the close-loop further down in the main() function.

Reply:
I agree with this, you can just write these functions inline.